apps/kg/train.py

```diff
@@ -134,13 +133,24 @@ def get_logger(args):
     return logger
 
 
+def get_compatible_batch_size(batch_size, neg_sample_size):
```

Comment:
Can we put this into some place that three files can share the same implementation?